dev/NumberBox/NumberBox.xaml

```diff
@@ -1,14 +1,13 @@
-﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
-<ResourceDictionary
```

Comment:
Almost all of those changes are just formatting. Can you please revert the formatting changes? 
With all these changes it is hard to review the actual new feature in question.

Reply:
I just reverted the changes. I had XAML Styler running which moved things around. Sorry.